Source/Classes/PINRemoteImageManager.m

```diff
@@ -36,10 +36,10 @@
 #endif
 
 
-#define PINRemoteImageManagerDefaultTimeout     30.0
-#define PINRemoteImageMaxRetries                3
-#define PINRemoteImageRetryDelayBase            4
-
+#define PINRemoteImageManagerDefaultTimeout            30.0
+#define PINRemoteImageMaxRetries                       3
+#define PINRemoteImageRetryDelayBase                   4
+#define PINRemoteImageHTTPMaximumConnectionsPerHost    20
```

Comment:
Does it make sense to set this to NSIntegerMax since it will be controlled by maxNumberOfConcurrentDownloads anyway?